content/docs/introduction/comparison.md

```diff
@@ -246,31 +246,30 @@ environment, then Prometheus is a good choice.
-
-There is also a [client socket](https://docs.sensu.io/sensu-core/latest/reference/clients/#what-is-the-sensu-client-socket) permitting ad-hoc check results to be pushed into Sensu. 
+Sensu processes events via the [Sensu Observablity Pipeline](https://docs.sensu.io/sensu-go/latest/observability-pipeline/), where events are deduplicated/correlated (via [EventFilters](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-filter/)), transformed (via [Mutators](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-transform/)), and processed (via [Handlers](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-process/)). Sensu also collects and routes metrics to a number of backend time-series databases such as Prometheus, InfluxDB, etc.
 
 ### Data model
 
-Sensu has the same rough data model as [Nagios](/docs/introduction/comparison/#prometheus-vs-nagios).
+Sensu primarily focuses on normalizing and processing observability data as [Events](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/), identified by an event name, entity (e.g. node, container, or service), and optional [key-value metadata](https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-events/events/#metadata-attributes) called "labels" or "annotations". The Sensu agents can also extract and route metrics from common formats such as Prometheus Exposition Format, InfluxDB line protocal, Graphite plain text, OpenTSDB line protocol, StatsD, and Nagios Perfdata and rewrite them into any backend storage format (Elastic, Influx, Wavefront, Prometheus).
 
 ### Storage
 
-Sensu uses Redis to persist monitoring data, including the Sensu client registry, check results, check execution history, and current event data.
+Sensu _caches_ event status information and real-time inventory data in an embedded database (etcd) or an external RDBMS (Postgres). All event and time series data is sent to one or more specialized datastores for long term storage and analysis (e.g. Prometheus, InfluxDB, Wavefront, TimescaleDB, Elasticsearch, Splunk).
 
 ### Architecture
 
-Sensu has a [number of components](https://docs.sensu.io/sensu-core/latest/overview/architecture/). It uses
-RabbitMQ as a transport, Redis for current state, and a separate server for
-processing and API access.
+Sensu is a primarily agent-based monitoring solution (agentless applications are also supported) ([reference](https://docs.sensu.io/sensu-go/latest/operations/deploy-sensu/deployment-architecture/)). The Sensu agents automatically connect to Sensu backends over an encrypted pub/sub transport (TLS-encrypted websockets), and handle automated registration/de-registration, and local service discovery. Sensu's use of a pub/sub transport allows it to securely work over complex network topologies including NATs, VPNs, and "air-gapped" environments (useful in IoT deployments).
```

Comment:
This section is to talk about clustering/HA, and right now only the very last sentence of this section covers that. So mention what the components are, and then that sentence up to the and.

The rest is not relevant.